types/react/index.d.ts

```diff
@@ -1160,7 +1160,7 @@ declare namespace React {
      *
      * target - a reference to the element from which the event was originally dispatched.
      * This might be a child element to the element on which the event listener is registered.
-     * If you thought this should be `EventTarget & T`, see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/12239
+     * If you thought this should be `EventTarget & T`, see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/11508#issuecomment-256045682
```

Comment:
```suggestion
     * If you thought this should be `EventTarget & T`, !see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/11508#issuecomment-256045682
```

what about, then, using the `@see` jsdoc directive (are they called directives? well whatever they're called)

Reply:
`@see` is not working inline. We actually want the word "see" here. 

https://www.typescriptlang.org/play/#code/JYOwLgpgTgZghgYwgAgApQPYAcDOyDeAUMicgPQBUFxpyFyAyhgLYqQAeYyAAjhCgAswYXAC4yZAObAwAgK4AjAHQIWZACIQYoGRAA2ATwAqBrBAAmGrTsiGTZy8Bw45EHGQCMHgKwAGABwAxE4uEKrMrOAAtABM3gBsvgAsCf4xNKQUZBnIYHBQkhBgoshwIAYA3DmU1LR0jCxsEJzIfILCYhLSsooqapraILp2phZWg8PGo47Oru5efkEhruGRYLEJyanpdVk5CHJQUBDgRvmFxaXlVQC+QA

Reply:
![Screenshot_20200623_121915](https://user-images.githubusercontent.com/15232461/85428827-c4f84100-b54b-11ea-8896-f679e1ef3910.png)

Works for me in VSCode, although they're both clickable so maybe no biggie.  Up to you then - just thought I'd mention a potential for improvement.  Apparently, today I learned, this is not a correct use of [`@see`](https://jsdoc.app/tags-see.html) anyway (it's intended for code symbols).  The correct tag looks to be [`@link`](https://jsdoc.app/tags-inline-link.html).  Either way (`@link` or `see`) is fine by me. :)